cypress/e2e/ScraperLiteEdition/Documentation.cy.js

```diff
@@ -0,0 +1,105 @@
+import { SCRAPER_BASE_URL, APILAYER_BASE_URL  } from '../../resources/data';
+ 
+describe('Scrapper(Lite Edition) Documentation page', () => {
+    before(() => {
+        cy.visit(SCRAPER_BASE_URL);
+    });
+ 
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+ 
+    it('2. Test to check "Documentation" tab visibility', () => {
+        cy.componentVisiblityCheck('.d-md-flex > .d-none');
+    });
+ 
+    it('3. Test to click "Documentation" tab', () => {
+        cy.get('.d-md-flex > .d-none').click();
```

Comment:
Add API interupt wait here. 
Also the tab has an id use that to get the element